config/config.yaml

```diff
@@ -15,10 +15,16 @@ cerberus:
     cerberus_publish_status: True                        # When enabled, cerberus starts a light weight http server and publishes the status
     inspect_components: False                            # Enable it only when OpenShift client is supported to run
                                                          # When enabled, cerberus collects logs, events and metrics of failed components
-    slack_integration: False                             # When enabled, cerberus reports status of each iteration in slack channel
+    slack_integration: False                             # When enabled, cerberus reports the failed iterations in slack channel
 
 tunings:
     iterations: 5                                        # Iterations to loop before stopping the watch, it will be replaced with infinity when the daemon mode is enabled
     sleep_time: 60                                       # Sleep duration between each iteration
     daemon_mode: True                                    # Iterations are set to infinity which means that the cerberus will monitor the resources forever
 
+cop_slack_ID:                                            # When slack_integration is enabled, a cop can be assigned for each day. The cop is tagged when failures are reported in a slack channel. Values are slack memeber ID.
```

Comment:
Maybe add a note that defining the cop id's is optional and when the cop slack id's are not defined, the failure messages are sent to the slack channel without tagging a specific person ( cop ) or uses "here" to tag everyone.